ndex2/client.py

```diff
@@ -1120,6 +1690,32 @@ def add_networks_to_networkset(self, set_id, networks):
     def delete_networks_from_networkset(self, set_id, networks, retry=5):
         """
         Removes network(s) from a network set.
+	
+	.. code-block:: python
+
+		import ndex2.client as nc
+		import ndex2
+		import networkx as nx
+		import pandas as pd
+		import os
+
+		set_id = 'INSERT SET UUID HERE'
+
+		#get information of the set, this method returns a dict 
+		network_set = my_ndex.get_network_set(set_id)
```

Comment:
Please add line showing how `my_ndex` is created. The only import needed is the one to import the Ndex2 client